### PR TITLE
Add Aqara Double Wall Switch US (WS-USC04)

### PR DESCRIPTION
**Pull request by niceboygithub**
Add Aqara Double Wall Switch US (WS-USC04)

### CODE REVIEW

**Comment by AlexxIT** [comment]
Thanks! I will merge request near to release version. There's a very big change right now.

File: custom_components/xiaomi_gateway3/core/gateway3.py [hunk 1]
```diff
@@ -1153,7 +1153,7 @@ def send(self, device: dict, data: dict):
                     'siid': int(k[0]), 'piid': int(k[2]), 'value': v
                 })
 
-            payload['miot_spec'] = params
+            payload['mi_spec'] = params
```
[the remark this answers is not in the document]
Comment (AlexxIT) [reply custom_components/xiaomi_gateway3/core/gateway3.py:1156]: Oh. You right! My mistake

**Comment by AlexxIT** [comment]
Thanks. Added to master from other branch

**Comment by lcsjandrey** [comment]
Tried pairing the switch but gateway beeps 3 times and deletes the device after a few seconds.

**Comment by AlexxIT** [comment]
Probably you should use another MiHome cloud region.

**Comment by lcsjandrey** [comment]
I’m using the Chinese server. I did try using an external converter but having a hard time with the documentation

**Comment by AlexxIT** [comment]
External converter wouldn't help with this situation. Because it is original `lumi` device. Gateway kick it off because regional restrictions. The component does not interact with the pairing process of the original devices.

**Comment by niceboygithub** [comment]
Cloud remove the switch US in cn region.
Nothing can do.

**Pull request closed without merging**